spec/index.bs

```diff
@@ -6708,6 +6751,16 @@ GPUQueue includes GPUObjectBase;
         </div>
 </dl>
 
+## Transfer operations ## {#transfer-operations}
+
+<dfn dfn>Transfers</dfn> is a class of operations on a queue that can be done for any {{GPUQueueKind}}:
+  - {{GPUQueue/writeBuffer()}}.
+  - {{GPUQueue/writeTexture()}}.
+  - {{GPUQueue/copyImageBitmapToTexture()}}.
```

Comment:
I'm not sure about this. There may be extra work for things like Y-flips, not sure if we can know they'll be doable (efficiently) on a transfer queue

Reply:
Added an issue nearby